pgjdbc/src/main/java/org/postgresql/core/v3/QueryExecutorImpl.java

```diff
@@ -367,6 +367,10 @@ private boolean sendAutomaticSavepoint(Query query, int flags) throws IOExceptio
         && query != restoreToAutoSave
         && !query.getNativeSql().equalsIgnoreCase("COMMIT")
         && getAutoSave() != AutoSave.NEVER
+        // Avoid savepoint does not exist error
```

Comment:
Should ignore case.
Reason: This would work well with the built-in syntax (con.setSavepoint() etc.) but not necessarily when the statement is used explicitly by the client (I do not mention compound statements or statements formatted with white space and/or comments).